include/z64actor.h

```diff
@@ -305,7 +305,9 @@ typedef struct Actor {
     /* 0x130 */ ActorFunc update; // Update Routine. Called by `Actor_UpdateAll`
     /* 0x134 */ ActorFunc draw; // Draw Routine. Called by `Actor_Draw`
     /* 0x138 */ ActorOverlay* overlayEntry; // Pointer to the overlay table entry for this actor
+#ifdef OOT_DEBUG
     /* 0x13C */ char dbgPad[0x10]; // Padding that only exists in the debug rom
+#endif
```

Comment:
we can maybe change "rom" to version, or just remove the comment altogether since now it will be obvious with the ifdef. Im ok with either

Reply:
Sure, is it because there is more than one debug version now? Or do you not like using the term "ROM" to refer to a specific version?

Reply:
yeah I think the term rom is out of place now. it made sense when our focus was decompiling one specific rom, but that isnt the case anymore. 
removing the comment might be best, cause it gets the same information across now. we arent going around commenting "debug version only" on all debug specific things. the ifdef already tells you :D